config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -0,0 +1,51 @@
+postsubmits:
+  kubernetes/org:
+  - name: post-org-run-peribolos
+    cluster: test-infra-trusted
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/cloud-builders/bazel
+        command:
+        - bazel
+        args:
+        - run
+        - //admin:update
+        - --
+        - --github-token-path=/etc/github-token/oauth
+        # TODO(fejta): --confirm
```

Comment:
So this would start running the job right after merge.. would there be log outputs in gubernator?

Reply:
Yes, see the above links

Reply:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/post-org-run-peribolos/1/?log#log

Reply:
Shows me for not reading the PR body haha